pkg/lwip/include/arch/sys_arch.h

```diff
@@ -125,13 +125,14 @@ void sys_lock_tcpip_core(void);
 void sys_unlock_tcpip_core(void);
 #define UNLOCK_TCPIP_CORE()        sys_unlock_tcpip_core()
 /** @} */
-
 #ifdef MODULE_RANDOM
 /**
  * @brief   Use `random_uint32()` to generate random numbers, if available
  */
+#if MBEDTLS_ENABLED == 0 
```

Comment:
Please fix the trailing whitespaces and the horizontal tabs mentioned by the automatic checks.